lib/Migration/RegisterBinary.php

```diff
@@ -67,14 +67,14 @@ public function run(IOutput $output): void {
 			}
 		}
 
-		// Make all binaries executable
-		chmod($binaryPath, 0755);
-
 		// Write the app config
 		$this->config->setAppValue('spreed', 'matterbridge_binary', $binaryPath);
 	}
 
 	protected function testBinary(string $binaryPath): ?string {
+        // Make binary executable
+        chmod($binaryPath, 0755);
```

Comment:
```suggestion
		// Make binary executable
		chmod($binaryPath, 0755);
```

Reply:
Whoops, IDE usually handles that. No idea what happened here